doc/classes/TileMap.xml

```diff
@@ -84,18 +117,47 @@
 		</method>
 		<method name="set_cell">
 			<return type="void" />
-			<argument index="0" name="coords" type="Vector2i" />
-			<argument index="1" name="source_id" type="int" default="-1" />
-			<argument index="2" name="atlas_coords" type="Vector2i" default="Vector2i(-1, -1)" />
-			<argument index="3" name="alternative_tile" type="int" default="-1" />
+			<argument index="0" name="layer" type="int" />
+			<argument index="1" name="coords" type="Vector2i" />
+			<argument index="2" name="source_id" type="int" default="-1" />
+			<argument index="3" name="atlas_coords" type="Vector2i" default="Vector2i(-1, -1)" />
+			<argument index="4" name="alternative_tile" type="int" default="-1" />
+			<description>
+			</description>
+		</method>
+		<method name="set_layer_enabled">
+			<return type="void" />
+			<argument index="0" name="layer" type="int" />
+			<argument index="1" name="enabled" type="bool" />
+			<description>
+			</description>
+		</method>
+		<method name="set_layer_name">
+			<return type="void" />
+			<argument index="0" name="layer" type="int" />
+			<argument index="1" name="name" type="String" />
 			<description>
-				Sets the tile index for the cell given by a Vector2i.
 			</description>
 		</method>
-		<method name="update_dirty_quadrants">
+		<method name="set_layer_y_sort_enabled">
 			<return type="void" />
+			<argument index="0" name="layer" type="int" />
+			<argument index="1" name="y_sort_enabled" type="bool" />
+			<description>
+			</description>
+		</method>
+		<method name="set_layer_y_sort_origin">
+			<return type="void" />
+			<argument index="0" name="layer" type="int" />
+			<argument index="1" name="y_sort_origin" type="int" />
+			<description>
+			</description>
+		</method>
+		<method name="set_layer_z_index">
+			<return type="void" />
+			<argument index="0" name="layer" type="int" />
+			<argument index="1" name="z_index" type="int" />
 			<description>
-				Updates the tile map's quadrants, allowing things such as navigation and collision shapes to be immediately used if modified.
```

Comment:
Lost docs.

Reply:
I removed this method. It's was supposed to be internal I guess.